ingest-app/src/cmr/ingest/services/granule_bulk_update/opendap/umm_g.clj

```diff
@@ -36,6 +37,14 @@
        first
        (update-opendap-url* (first (opendap-type url-map)))))
 
+(defn- updated-type-related-url
+  "Updates types"
+  [related-url]
+  (if (re-find #"opendap" (string/lower-case (:URL related-url)))
```

Comment:
Curious what @eereiter thinks here - I opted to identify any link containing the string `opendap` as an opendap link. Meaning, any link that matches will have its Type and Subtype updated.

Are there any alternatives we should consider? Could this be overzealous in identifying opendap links?